src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAlignment.java

```diff
@@ -0,0 +1,174 @@
+            final AlignmentRegion alignmentRegion1 = AlignmentRegion.fromString(alignmentRegion1Fields);
+            final String[] alignmentRegion2Fields = Arrays.copyOfRange(fields, 10, 19);
+            final AlignmentRegion alignmentRegion2 = AlignmentRegion.fromString(alignmentRegion2Fields);
+            final String insertedSequence = fields[19].equals(NO_SEQUENCE) ? "" : fields[19];
+            final String homology = fields[20].equals(NO_SEQUENCE) ? "" : fields[20];
+            final List<String> insertionMappings = Arrays.asList(fields[21].split(";"));
+            return new BreakpointAlignment(contigId, alignmentRegion1, alignmentRegion2, insertedSequence, homology, insertionMappings);
+        } catch (final NumberFormatException nfe) {
+            throw new GATKException(Arrays.toString(fields), nfe);
+        }
+
+    }
+
+    private AlignmentRegion getLeftAlignmentRegion() {
+        if (region1.referenceInterval.getStart() < region2.referenceInterval.getStart()) {
+            return region1;
+        } else {
+            return region2;
+        }
+    }
+
+    @VisibleForTesting
+    SimpleInterval getLeftAlignedLeftBreakpointOnAssembledContig() {
+        if (region1 == getLeftAlignmentRegion()) {
+            final int position = region1.forwardStrand ? region1.referenceInterval.getEnd() - homology.length() : region1.referenceInterval.getStart();
+            return new SimpleInterval(region1.referenceInterval.getContig(), position, position);
+        } else {
+            final int position = region2.forwardStrand ? region2.referenceInterval.getStart() : region2.referenceInterval.getEnd() - homology.length();
+            return new SimpleInterval(region1.referenceInterval.getContig(), position, position);
+        }
+    }
+
+    @VisibleForTesting
+    SimpleInterval getLeftAlignedRightBreakpointOnAssembledContig() {
+        if (region1 == getLeftAlignmentRegion()) {
+            final int position = region2.forwardStrand ? region2.referenceInterval.getStart() + homology.length() : region2.referenceInterval.getEnd();
+            return new SimpleInterval(region2.referenceInterval.getContig(), position, position);
+        } else {
+            final int position = region1.forwardStrand ? region1.referenceInterval.getEnd() - homology.length() : region1.referenceInterval.getStart();
+            return new SimpleInterval(region2.referenceInterval.getContig(), position, position);
+        }
+    }
+
+    /**
+     * Returns the canonical representation of the breakpoint implied by this split contig alignment,
+     * including whether it is a 3-5 or 5-3 inversion, and the homology and inserted sequence at the
+     * breakpoint. The two intervals returned are 1bp intervals indicating the exact breakpoint
+     * location. If there is homology at the breakpoint, the breakpoint locations will be left
+     * aligned.
+     * @return
+     */
+    public BreakpointAllele getBreakpointAllele() {
```

Comment:
That would be a valid choice but I'm not sure I see a compelling reason to switch it right now.